shared/settings/account/email-phone-row.tsx

```diff
@@ -191,14 +201,14 @@ const mapDispatchToProps = (dispatch: Container.TypedDispatch, ownProps: OwnProp
   _onMakeSearchable: () =>
     dispatch(SettingsGen.createEditEmail({email: ownProps.contactKey, makeSearchable: true})),
   email: {
-    _onDelete: (address: string, discoverable: boolean) =>
+    _onDelete: (address: string, searchable: boolean) =>
       dispatch(
         RouteTreeGen.createNavigateAppend({
           path: [
             {
               props: {
                 address,
-                discoverable,
```

Comment:
`settingsDeleteAddress` expects this to be `searchable`, this fixes the plumbing.